jetbrains-gateway/src/software/aws/toolkits/jetbrains/gateway/CawsConnectionProvider.kt

```diff
@@ -231,6 +234,14 @@ class CawsConnectionProvider : GatewayConnectionProvider {
                                     }
                                 }
                             }
+                            CodecatalystTelemetry.devEnvironmentWorkflowStatistic(
+                                project = null,
+                                userId = userId,
+                                result = if (attemptCount == 15) TelemetryResult.Failed else TelemetryResult.Succeeded,
+                                duration = (System.currentTimeMillis() - start).toDouble(),
+                                codecatalystDevEnvironmentWorkflowStep = "fileSystem Check",
+                                numberOfRetryAttempts = (attemptCount).toString()
```

Comment:
we could also use the `value` field to capture this data

Reply:
Sure, will be implemented using value filed